extensions/interactions/MathEquationInput/directives/math-equation-input-validation.service.ts

```diff
@@ -40,6 +40,8 @@ import { NumericExpressionInputRulesService } from 'interactions/NumericExpressi
   providedIn: 'root'
 })
 export class MathEquationInputValidationService {
+  private supportedFunctionNames = AppConstants.SUPPORTED_FUNCTION_NAMES;
```

Comment:
Why is this needed?

Reply:
This is used to show the list of supported functions via the validation error message.